### ISSUE TITLE
Finalize tool name

### ISSUE THREAD

**Issue by lrettig** [spacemeshos/smcli#48]
The CLI command (the binary) was originally `sm` but it's now `smcli`. The help message is therefore contradictory:

```
> ./smcli -h
sm is your terminal's connection to the Spacemesh network.

The sm CLI provides an ergonomic set of tools for managing a node and wallet.

Usage:
  sm [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  wallet      A brief description of your command

Flags:
      --config string   config file (default is $HOME/.smcli.yaml)
  -h, --help            help for sm
  -t, --toggle          Help message for toggle

Use "sm [command] --help" for more information about a command.
```

Pick a name and harmonize these.

**Comment by 0xjac** [comment]
@lrettig Happy to fix it.
Do you want to go the fancy way with [`debug.BuildInfo`](https://pkg.go.dev/runtime/debug#BuildInfo) to retrieve the module name auto-magically? Or do you prefer a constant which is used across the project (and can be easily overwritten when compiling)?

Also if you are courageous about picking a name, I think `smctl`, "à la systemd" (`systemctl`, `journalctl`, etc.), is nicer than `smcli`. :grimacing: :wink:

**Comment by lrettig** [comment]
@0xjac thanks for offering to help here! I think before renaming this we need to figure out its destiny -- is it simply going to be a CLI wallet (which it is now), or will it be a "swiss army knife" with lots of other features? CC @spacemeshos/core-dev-team 

If the former, `smctl` probably isn't the best name. If the latter then that's a great choice.

**Comment by 0xjac** [comment]
Yeah, I understood this tool more as the latter than the former but up to you. In the meantime I am happy to just provide a fix where the current name (`smcli`) is defined in a constant and used across the code base. Making it easy to change the name later either in the code or at compilation.

Would that be ok for you?

**Comment by countvonzero** [comment]
@0xjac i think that's a great idea. feel free to put me as a reviewer. thanks
